File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/Result.java
package com.vladislavmyasnikov.courseproject.data.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Result {

    @SerializedName("homeworks")
    private List<Lecture> mLectures;

    @SerializedName("user")
    private User mUser;

    public User getUser() {
        return mUser;
    }

    public List<Lecture> getLectures() {
        return mLectures;
    }
}

File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/Result.kt
package com.vladislavmyasnikov.courseproject.data.models

import com.google.gson.annotations.SerializedName

class Result {

    @SerializedName("homeworks")
    val lectures: List<Lecture>? = null

    @SerializedName("user")
    val user: User? = null

    @SerializedName("grades")
    val students: List<Student>? = null
}
